packages/e2e-test/src/commands/index.ts

```diff
@@ -18,5 +18,13 @@ import { Command } from 'commander';
 import { run } from './run';
 
 export function registerCommands(program: Command) {
-  program.command('run').description('Run e2e tests').action(run);
+  program
+    .command('run')
+    .description('Run e2e tests')
+    .option(
+      '-db, --dbms <value>',
```

Comment:
Do we need this flag? The existing behavior was to be predicated on the presence of the `POSTGRES_USER` variable; we could remove this option and predicated on the presence of either one of two variables instead. Otherwise we kinda have two mechanisms for the same thing, especially since there's currently a default value set on this one

Reply:
We don't need this, I was experimenting different methods to check, good catch